### PR TITLE
[13.0][MIG]website_crm_quick_answer: Migration to 13.0

### PR DESCRIPTION
**Pull request by HaraldPanten**

### CODE REVIEW

**Comment by HaraldPanten** [comment]
Hi @pedrobaeza @Yajo @ValentinVinagre 

Could you review? THX!

**Review by manuelregidor — APPROVED** [review]
Working fine.

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍

File: website_crm_quick_answer/README.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwebsite-lightgray.png?logo=github
+    :target: https://github.com/OCA/website/tree/12.0/website_crm_quick_answer
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/README.rst]:
```suggestion
    :target: https://github.com/OCA/website/tree/13.0/website_crm_quick_answer

```

File: website_crm_quick_answer/README.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/website <https://github.com/OCA/website/tree/12.0/website_crm_quick_answer>`_ project on GitHub.
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/README.rst]: 13.0

File: website_crm_quick_answer/data/base_automation_data.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <data>
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/data/base_automation_data.xml]: @HaraldPanten no need to `<data>`, please remove everywhere

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
nitpicking

File: website_crm_quick_answer/__manifest__.py [hunk 4]
```diff
@@ -1,15 +1,16 @@
 # Copyright 2016 Tecnativa - Jairo Llopis
 # Copyright 2017 Tecnativa - David Vidal
 # Copyright 2019 Tecnativa - Cristina Martin R.
+# Copyright 2020 Sygel Technology - Harald Panten
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/__manifest__.py]: @HaraldPanten Please remove your Copyright, you are not an `author`. Only you add a name on the CONTRIBUTORS section.

Comment (HaraldPanten) [reply website_crm_quick_answer/__manifest__.py]: I thought I had to add myself as I applied some changes. For sure I do not consider myself as an author, but only another contributor.

@pedrobaeza could you shed some light here?

**Comment by HaraldPanten** [comment]
Hi @nikul-serpentcs could you check?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by HaraldPanten** [comment]
@pedrobaeza I think this one is ready to be merged 👍

**Pull request merged**